src/nvim/quickfix.c

```diff
@@ -1772,7 +1772,7 @@ static void decr_quickfix_busy(void)
 void check_quickfix_busy(void)
 {
   if (quickfix_busy != 0) {
-    EMSGN("quickfix_busy not zero on exit: %ld", (long)quickfix_busy);
+    EMSGN("quickfix_busy not zero on exit: %+" PRId64, (int64_t)quickfix_busy);
```

Comment:
what does the `+` do? launch the nukes?

Reply:
Oopf, this is incorrect. `%+` prints out a '+' in front of the value (like a positive integer in printf).

Once more into the void 😓